CMakeLists.txt

```diff
@@ -213,42 +214,6 @@ if(CORE_SYSTEM_NAME STREQUAL osx)
   core_require_dep(Sdl)
 endif()
 
-if(NOT WIN32)
```

Comment:
This whole block is not linux only, so for the other platforms we either need to put these dependencies in the ArchSetup files, or also create platform files for them. If there's a change that we will have more platforms also for non-linux then the later is probably preferable. (Maybe @FernetMenta / @Memphiz / @Paxxi can give input about how likely that is).